src/sql/util.py

```diff
@@ -157,6 +157,113 @@ def show_deprecation_warning():
     )
 
 
+def check_duplicate_arguments(
+    magic_execute, cmd_from, args, allowed_duplicates=[]
+) -> bool:
+    """
+    Raises UsageError when duplicate arguments are passed to magics.
+    Returns true if no duplicates in arguments or aliases.
+
+    Parameters
+    ----------
+    magic_execute
+        The execute method of the magic class.
+    cmd_from
+        Which magic class invoked this function. One of 'sql', 'sqlplot' or 'sqlcmd'.
+    args
+        The arguments passed to the magic command.
+    allowed_duplicates
+        The duplicates that are allowed for the class which invoked this function.
+        Defaults to an empty list.
+
+    Returns
+    -------
+    boolean
+        When there are no duplicates, a True bool is returned.
+    """
+
+    # Returns True if there are no duplicate arguments (1 or less arguments)
+    # Example: %sqlcmd tables
+    if len(args) <= 1:
```

Comment:
It looks like this function is called when len is > 1:
```
if len(args) > 1:
        check_duplicate_arguments(magic_execute, cmd_from, args, allowed_duplicates)
```

Why do we need this condition then?

Reply:
'check_duplicate_arguments' is also called directly from the SqlCmdMagic class. It does not call the parser. So I added this condition here as a sanity check and to ensure that cases where `sqlcmd` arguments are less than or equal to 1 return `True`.

Reply:
Removed the need for having that check in [this](https://github.com/ploomber/jupysql/pull/908/commits/d466890f40cf69bd48a2ea8bf0c9b7c6719eee40) commit using [this](https://github.com/ploomber/jupysql/pull/908/files#diff-2aa72d6ecafc2697ec8174353868dae816494131c70b7e48be6115fda9f61370R78-R79) check in the `SqlCmdMagic` class.